ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModule.java

```diff
@@ -539,6 +551,37 @@ public void sendAccessibilityEvent(int tag, int eventType) {
     mUIImplementation.sendAccessibilityEvent(tag, eventType);
   }
 
+  public void emitUpdateDimensionsEvent() {
+    mFontScale = getReactApplicationContext().getResources().getConfiguration().fontScale;
```

Comment:
imo, this shouldn't be in emitUpdate() given the name. I'd just set it in onHostResume